src/constants/jsonRpcEndpoints.ts

```diff
@@ -86,4 +86,14 @@ export const JSON_RPC_FALLBACK_ENDPOINTS: Record<SupportedChainId, string[]> = {
     'https://bsc-dataseed4.defibit.io',
     'https://rpc.ankr.com/bsc',
   ],
+  [SupportedChainId.BASE]: [
+    // "Safe" URLs
+    'https://mainnet.base.org',
+    'https://developer-access-mainnet.base.org/',
```

Comment:
Lets remove `developer-access-mainnet.base.org`, it resolves to the same hosts as `mainnet.base.org` and at some point we may remove it.